nativelib/src/main/scala/scala/scalanative/runtime/PlatformExt.scala

```diff
@@ -0,0 +1,10 @@
+package scala.scalanative.runtime
+
+import java.util.Locale
+
+object PlatformExt {
+  private lazy val osArch =
+    System.getProperty("os.arch").toLowerCase(Locale.ROOT)
```

Comment:
Scala Native does not define `java.util.Locale`, but it can be provided via external libraries. For usage inside Scala Native us just `.toLowerCase` it should be compliant with usage of `Locale.ROOT`